diff --git a/libupnpp/upnpp_p.hxx b/libupnpp/upnpp_p.hxx
index 073de2b..f6c191b 100644
--- a/libupnpp/upnpp_p.hxx
+++ b/libupnpp/upnpp_p.hxx
@@ -31,7 +31,7 @@
        (UPNP_VERSION_MINOR == (B) && UPNP_VERSION_PATCH >= (C)))))
 
 #if PUPNP_AT_LEAST(1,8,0) && !PUPNP_AT_LEAST(1,8,5)
-#error PUPNP versions between 1.8.0 and 1.8.4 (included) do not work
+//#error PUPNP versions between 1.8.0 and 1.8.4 (included) do not work
 #endif
 
 #include <pthread.h>
